profiles/himanshu1221.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/32031706?v=4",
```

The image must be the image link provided in the issue you created (#1698).
```suggestion
  "image": "https://user-images.githubusercontent.com/32031706/216389505-bff1996b-1553-4867-bc8e-05eba9aabb6a.png",
```

---

okay :+1: